crates/iota/src/unit_tests/profiler_tests.rs

```diff
@@ -62,6 +61,7 @@ async fn test_profiler() {
     let command_result =
         iota_replay::execute_replay_command(Some(testnet_url), false, false, None, None, cmd).await;
 
+    println!("{:?}", command_result);
```

Comment:
Do you want to keep it or was it just for debugging purposes?

Reply:
It was just for debugging purposes
I’ve removed it